web_src/js/utils.js

```diff
@@ -128,3 +128,14 @@ export function decodeURLEncodedBase64(base64url) {
     .replace(/_/g, '/')
     .replace(/-/g, '+'));
 }
+
+const parser = new DOMParser();
+const serializer = new XMLSerializer();
+
+export function parseXml(text, contentType) {
```

Comment:
```suggestion
const domParser = new DOMParser();
const xmlSerializer = new XMLSerializer();

export function parseDom(text, contentType) {
```

The old name `parseXml` is inaccurate. According to MDN: it parses `XML, SVG, and HTML`

Reply:
All three of those are XML variants.

Reply:
But we all know that HTML is not valid XML. XML doesn't render HTML.

And HTML standard groups do not call `DOMParser` as `XMLParser`.

Reply:
Fixed.